lib/cosmos/dart/lib/dart_common.rb

```diff
@@ -444,6 +444,10 @@ def create_table(table_name)
     # configuration we drop anything that may be existing.
     if ActiveRecord::Base.connection.table_exists?(table_name)
       ActiveRecord::Base.connection.drop_table(table_name)
+      # Once we drop an existing table we have to reset_column_information
+      # or we'll just get back the column names from the original table
+      model = Cosmos.const_get(table_name.capitalize.intern)
+      model.reset_column_information
```

Comment:
This was another fun one. Once ActiveRecord loads a table it assumes it's not going to change. Since we're dropping and re-creating a table here the columns were still the old table. Not sure if this would have been an issue in the real DART system but it hoses up testing.